llvm/include/llvm/IR/Function.h

```diff
@@ -996,6 +1012,12 @@ class LLVM_EXTERNAL_VISIBILITY Function : public GlobalObject,
   void setValueSubclassDataBit(unsigned Bit, bool On);
 };
 
+#ifdef NDEBUG
+/// In release builds, this is a no-op. For !NDEBUG builds, the checks are
+/// implemented in the .cpp file.
+inline void Function::validateBlockNumbers() const {}
+#endif
```

Comment:
I'd prefer not to have the conditional definitions. Especially as the only callers are currently in the same file, there is no penalty to having the ifdef NDEBUG over the function body only.

Reply:
Agreed, changed